desktop/apps/artwork/components/artists/helpers.coffee

```diff
@@ -15,21 +15,24 @@ module.exports =
   sortExhibitions: (shows) ->
     sortBy(shows, 'start_at').reverse()
 
-  sections: sections = (artist) ->
+  sections: sections = (artist, isAdmin) ->
```

Comment:
This change is for the comp: admins always see an exhibition history tab, and not just when there's too much to display inline (since we're not displaying it inline anymore either).